build_defs/internal/kotlin.bzl

```diff
@@ -36,20 +36,49 @@ def kt_jvm_and_js_library(
     kt_jvm_library(
         name = name,
         srcs = srcs,
-        deps = deps,
+        deps = [_to_jvm_dep(dep) for dep in deps],
```

Comment:
I'm having DejaVu :) (similar rules were just deleted for simultaneous JVM/WASM targets)